lib/rubocop/cop/style/empty_case_condition.rb

```diff
@@ -77,10 +77,14 @@ def correct_multiple_alternative_whens(corrector, when_nodes)
 
             # In `when a; r` we have two children: [a, r].
             # In `when a, b, c; r` we have 4.
-            next unless children.count > 2
+            next unless children.size > 2
```

Comment:
Nice, I didn't notice that when I wrote it originally! (I wonder if there should be a cop for such a thing! :-) )